src/Tests/AddressTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+declare(strict_types=1);
+
+namespace GingerPluginSdk\Tests;
+
+use GingerPluginSdk\Entities\Address;
+use GingerPluginSdk\Properties\Country;
+use GingerPluginSdk\Exceptions\OutOfEnumException;
+use GingerPluginSdk\Exceptions\OutOfPatternException;
+use GingerPluginSdk\Helpers\SingleFieldTrait;
+use PHPUnit\Framework\TestCase;
+use TypeError;
+
+class AddressTest extends TestCase
+{
+    private Address $address;
+
+    public function __construct(?string $name = null, array $data = [], $dataName = '')
+    {
+        $this->address = new Address(
+            address_type: "customer",
+            postal_code: "38714",
+            street: "Soborna",
+            city: "Poltava",
+            country: new Country(
+                "UA"
+            )
+        );
+        self::assertSame((bool)$this->address, true);
```

Comment:
This assertions does not test anything. You want to test the observable behaviour. I assume in this case you want to test for the response of the toArray method.

Reply:
Ye, good remark